src/DS2434.cpp

```diff
@@ -148,38 +168,38 @@ void DS2434::setTemperature(const int8_t temp_degC)
 
     if (value > 126) value = 126;
     if (value < -40) value = -40;
-    memory[61] = value;
+    memory[0x61] = value;
```

Comment:
good spot! cleary a (repeating) bug in this file :(

Reply:
Ok, cool. Wanted to make sure I wasn't missing something!